app/helpers/cornell_catalog_helper.rb

```diff
@@ -1432,31 +1432,27 @@ def sub_tilde_path(var,id,aeon_codes,document)
   end
 
 # (group == "Circulating" ) ? blacklight_cornell_request.magic_request_path("#{id}") :  "http://wwwdev.library.cornell.edu/aeon/monograph.php?bibid=#{id}&libid=#{aeon_codes.join('|')}"
-  def request_path(group,id,aeon_codes,document,scan)
-	id_scan = scan == "yes" ? "#{id}.scan" : "#{id}"
+  def request_path(group, id, aeon_codes, document, scan)
+    id_scan = scan ? "#{id}.scan" : "#{id}"
     magic_path  = blacklight_cornell_request.magic_request_path("#{id_scan}")
+
     if ENV['SAML_IDP_TARGET_URL'].present?
-    magic_path = blacklight_cornell_request.auth_magic_request_path("#{id_scan}")
+      magic_path = blacklight_cornell_request.auth_magic_request_path("#{id_scan}")
     end
+
     if ENV['AEON_REQUEST'].blank? and group != 'AEON_SCAN_REQUEST'
```

Comment:
Optional, but maybe we could also change `and` to `&&`? Reduces possibility of error in any future refactors due to confusingly different levels of precedence in ruby.

Reply:
Good idea!